doc/authors.md

```diff
@@ -55,21 +57,24 @@ aniemerg (2)
 UdjinM6 (2)
 Scott (2)
 Robert C. Seacord (2)
+Pejvan (2)
 Pavol Rusnak (2)
 Pavel Vasin (2)
 Matthew King (2)
-Larry Ruane (2)
+Kaz Wesley (2)
 Joe Turgeon (2)
 Jack Gavigan (2)
 ITH4Coinomia (2)
 Gavin Andresen (2)
 Brad Miller (2)
 Bjorn Hjortsberg (2)
 Amgad Abdelhafez (2)
+Alex Morcos (2)
 zathras-crypto (1)
 unsystemizer (1)
 practicalswift (1)
 mrbandrews (1)
+mdr0id (1)
```

Comment:
This should be added to the canonicalization list if @mdr0id wants his full name here. (Does not block.)